packages/xod-client-electron/src/app/arduinoDependencies.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Module for Main Process
+ * about checking and installing arduino dependencies:
+ * - libraries
+ * - toolchains (TODO #1201)
+ * - arduino packages (TODO #1201)
+ *
+ * It incapsulates all IPC communication between Main Process
+ * and Renderer Process and checks/installs dependencies.
```

Comment:
:+1: for the module description. But why “the main process” and “the rendering process” are capitalized? There’s “the” to express exactly what you mean as far as I understand the intent.